src/app/styles/normalize.scss

```diff
@@ -24,21 +25,10 @@ code {
 }
 
 ::-webkit-scrollbar {
-    width: 8px;
-    height: 8px;
-    background-color: transparent;
-
-    &-track {
-        background-color: transparent;
-        border-radius: 5px;
-    }
-
-    &-thumb {
-        background-color: #a6b2c3;
-        border-radius: 5px;
-    }
-
-    &-corner {
-        background-color: #ffffff;
-    }
-}
\ No newline at end of file
+    width: 0; 
```

Comment:
Please, do not change normilize styles. Create different file and there rewrite scrollbar.